client/main.lua

```diff
@@ -294,17 +299,22 @@ AddStateBagChangeHandler('spotlight', nil, function(bagName, _, value)
 end)
 
 lib.onCache('seat', function(seat)
-    if seat ~= -1 and seat ~= 0 then
-        spotlight:disable(true)
-        return
-    end
-
     local model = GetEntityModel(cache.vehicle)
 
-    if not config.policeHelicopters[model] then return end
+    if not config.authorizedHelicopters[model] then return end
+
+    if seat == -1 or seat == 0 then
+        rappel:disable(true)
 
-    if DoesVehicleHaveSearchlight(cache.vehicle) then
-        spotlight:disable(false)
+        if DoesVehicleHaveSearchlight(cache.vehicle) then
+            spotlight:disable(false)
+        end
+    elseif seat == 1 or seat == 2 then
+        spotlight:disable(true)
+
+        if DoesVehicleAllowRappel(cache.vehicle) then
+            rappel:disable(false)
+        end
```

Comment:
what about helicopters which fit more players? Should this be seat >= 1?

Reply:
I wrote this thinking that they should have to be on the door seats to be able to rappel out as it seemed weird for people to be able to do it from anywhere else. Do you think they should be able to do it anywhere besides the front two seats?

Reply:
Yeah I do as it's pretty cumbersome to force players to switch to the door seats to rappel, even though it makes sense.

Reply:
Sounds good. I'll push the change